src/components/applications/ApplicationDetails.js

```diff
@@ -0,0 +1,126 @@
+import {ListElementDetails} from "../commons/ListElementDetails";
+import {Box, Button, CardContent, Divider, Grid, Typography} from "@material-ui/core";
+import {ApplicationTimeline} from "./ApplicationTimeline";
+import {FileViewerWrapper} from "./FileViewerWrapper";
+import {withSwal} from "../commons/formsCommons/WithSwal";
+import {applicationsAPI} from "../../utils/apis/applicationsAPI";
+import Swal from "sweetalert2";
+
+export const ApplicationDetails = ({application, isHR}) => {
+
+    console.log(application)
```

Comment:
hihi xd

Reply:
A ja Ci nawet powiem skąd mi to zostało. Bo to była ta historia z nowym applicationDTO i chciałem zobaczyć, co tam dostaje XDDD
Plus pewnie i tak się przyda na razie zostawić, bo coś się te stage nie zgadzają